packages/contracts/contracts/test/ValidatorManager.t.sol

```diff
@@ -613,12 +615,17 @@ contract ValidatorManagerTest is ValidatorSystemUpgrade_Initializer {
     function test_finalizeCommissionChange_exited_reverts() external {
         _registerValidator(trusted, minActivateAmount);
 
-        uint128 kroShares = assetMgr.getKroTotalShareBalance(trusted, trusted);
         vm.prank(trusted);
-        assetMgr.initUndelegate(trusted, kroShares);
+        valMgr.initCommissionChange(15);
+
+        uint128 validatorKro = assetMgr.totalValidatorKro(trusted);
+        vm.warp(assetMgr.canWithdrawAt(trusted) + 1);
+        vm.startPrank(withdrawAcc);
+        assetMgr.withdraw(trusted, validatorKro);
```

Comment:
How about add _withdraw internal function to `ValidatorManagerTest` contract like `_setUpHundredKghDelegation`? These lines are repeated several times.

Reply:
Yeah, I made _withdraw as you suggested.

Reply:
Can you replace L333-335 also?

Reply:
I missed it. Thanks